glotpress.php

```diff
@@ -27,7 +27,7 @@
  */
 
 define( 'GP_VERSION', '2.4.0-alpha' );
-define( 'GP_DB_VERSION', '980' );
+define( 'GP_DB_VERSION', '981' );
```

Comment:
I should have mentioned, we usually bump the number in increments of 10, don't ask me why, just the way it's been done in the past.

Reply:
fixed